package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "boilerplate",
+  "version": "1.0.0",
+  "description": "Boilerplate for node.js based micro service",
+  "main": "index.js",
+  "repository": "git@github.com:plyo/boilerplate.git",
+  "author": "Andrew Balakirev <balakirev.andrey@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "cookie-parser": "^1.4.3",
+    "express": "^4.16.3",
+    "pino": "^4.17.3",
+    "pino-elasticsearch": "^2.2.1",
+    "pino-http": "^3.2.2",
+    "superagent": "^3.8.3"
```

Comment:
Could you please provide some examples of microservices? Does a typical microservice send http requests?

Reply:
Yes, for us it usually microservices which communicate by HTTP. First free real tasks are integrations with 3rd party API. The services communicate with these APIs periodically, format answers special way and send to our API.

Reply:
Thanks!